domains/transfer/Transfer.API/API/Transfer/Api/Dto/Requests/CreateTransferAgreementProposal.cs

```diff
@@ -1,12 +1,36 @@
 using System;
+using DataContext.Models;
 using FluentValidation;
+using System.Text.Json.Serialization;
 
 namespace API.Transfer.Api.Dto.Requests;
 
 public record CreateTransferAgreementProposal(
     long StartDate,
     long? EndDate,
-    string? ReceiverTin);
+    string? ReceiverTin,
+    CreateTransferAgreementType? Type = null);
```

Comment:
CreateTransferAgreementType? Type = TransferAllCertificates?? so default value is all?